registration/registration-services/src/main/java/io/mosip/registration/entity/UserBiometric.java

```diff
@@ -30,13 +30,17 @@ public class UserBiometric extends RegistrationCommonFields {
 	private UserBiometricId userBiometricId;
 	
 	@Lob
-	@Column(name = "bio_raw_image")
-	private byte[] bioRawImage;
+	@Column(name = "bir_data")
```

Comment:
As we use CbeffValidator.getBIRFromXML() to deserialize, and this is a common library used across modules, so change in XSD format means backward comaptibility code should be part of kernel-cbeffutil-api library.
This means we don't have to store the BIR version. lets use the existsing bio_raw_image column itself.
Please add a comment here, about what we store in that column before and after 1.2.0.1-B2 release(documentation) .

Reply:
Modified as suggested and gave comments